recipe/build.sh

```diff
@@ -27,16 +27,16 @@ export CXX_host=$CXX_FOR_BUILD
 export AR_host=$($CC_FOR_BUILD -print-prog-name=ar)
 export LDFLAGS_host="$(echo $LDFLAGS | sed s@${PREFIX}@${BUILD_PREFIX}@g)"
 
-./configure \
+"${SRC_DIR}"/configure \
     ${EXTRA_ARGS} \
     --ninja \
     --prefix=${PREFIX} \
     --shared \
     --shared-libuv \
     --shared-openssl \
     --shared-zlib \
-    --with-intl=none
-    # --with-intl=system-icu
+    --with-intl=none \
```

Comment:
```suggestion
```

We do want internationalization, it was removed before because our icu was too old.
The comment line 40/41 should be removed as well.